src/containers/AlertGeneral/AlertGeneral.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+export class AlertGeneral extends Component {
```

Comment:
Since this is a presentational component, could be a good case to use a function component which only receives the props object to reduce the boilerplate of this module.